### DISCUSSION TITLE
Floppy disk Icon / Save Icon

### DISCUSSION THREAD

**Discussion by abcmoritz** [tailwindlabs/heroicons#851] [Ideas]
Hey there,
I like the collection and style of the heroicons.
But Im really missing a save icon. Is there a reason why the famous floppy disk is not integrated?

**Comment by skstuder** [comment]
+1, love the library, I have a use case that requires a floppy disk.

**Comment by simonebeets** [comment]
The only reason we're now not using heroicons is because we need a save icon - this icon would be useful to have.

**Comment by ndri** [comment]
I added `floppy-disk` to [Sidekickicons](https://github.com/ndri/sidekickicons/), which are meant to be used as a companion to Heroicons. Browse them here: https://sidekickicons.com/?icon=floppy-disk

**Comment by Awesomolocity** [comment]
I'm actually shocked that the library doesn't include the floppy disk "save" icon. It's so universal that I just never thought to even look. It feels almost like an oversight, since the icon exists in Zondicons.
